src/components/play/page/PagePlayContent.tsx

```diff
@@ -172,12 +164,10 @@ const PagePlayContent: React.FC<PagePlayContentProps> = (
     ]);
 
     return (
-        <FocusedElement>
```

Comment:
FocusedElement was removed because it turns out it wasn't needed actually. probably wasn't needed once the key handlers were mounted on the document instead of the body element.